package.json

```diff
@@ -27,12 +27,15 @@
     "@types/express": "^4.17.17",
     "@types/jest": "^29.5.1",
     "@types/node": "^20.1.4",
+    "@types/pg": "^8.10.1",
     "@types/prettier": "^2.7.2",
+    "@types/supertest": "^2.0.12",
     "@typescript-eslint/eslint-plugin": "^5.59.5",
     "@typescript-eslint/parser": "^5.59.5",
     "eslint": "^8.40.0",
     "jest": "^29.5.0",
     "prettier": "^2.8.8",
+    "supertest": "^6.3.3",
```

Comment:
We don't use this right?

Reply:
> We don't use this right?

We don't use it right now Anjula.But when we write test code for endpoints I think we have to use it.
May I remove it?

"If you're writing tests for an HTTP server or API endpoints, supertest can be a useful library to use in combination with jest. "

Reply:
let's remove for now

Reply:
@janithlahirukariyawasam let's remove this